core/templates/domain/utilities/language-util.service.ts

```diff
@@ -157,30 +157,27 @@ export class LanguageUtilService {
     return language ? language.description : null;
   }
 
-  getContentLanguageDescription(contentLanguageCode: string): string {
-    const language = this.getSupportedContentLanguages()[contentLanguageCode];
-    return language ? language.description : null;
-  }
+  getLanguageDirection(languageCode: string): string {
+    // Any overlap between Content Languages and Audio Languages should be
+    // identical. Each list should contain one entry per language code.
```

Comment:
Is this last requirement validated in a frontend test? If not, do so and change this to something like "The Karma tests guarantee that each list contains at most one entry per language code." (Similarly for the first sentence.)

This helps make clear that what you're saying is that "these lists should already adhere to these criteria" and not "the developer should ensure that this should happen".

Reply:
I added additional tests and update the language as described.